rpc/client.ts

```diff
@@ -103,7 +114,13 @@ export class Client<
         );
       },
     }) as ClientSubscribeListener<SendErrorData, HandlerErrorData>;
-    this.call(message);
+    this.call(message)
+      .then((maybeError) => {
+        if (maybeError instanceof Error) {
+          this.pendingSubscriptions[message.id]!(maybeError);
```

Comment:
Perhaps alias the `listener.bind...` block on 106 as `listenerBound` before assigning to `this.pendingSubscriptions[message.id]` (`message.id` could also be aliased within this method as `id`)?